pynet/datasets/hcp.py

```diff
@@ -0,0 +1,104 @@
+import os
+from collections import namedtuple
+
+import boto3
```

Comment:
boto3 & nibabel are now dependencies of the package: add them in pynet/info.py & doc/requirements.txt.